samples/vk_hellotriangle.cpp

```diff
@@ -54,10 +54,12 @@ static const Vertex TRIANGLE_VERTICES[3] = {
 
 static constexpr uint16_t TRIANGLE_INDICES[3] = { 0, 1, 2 };
 
-static constexpr uint8_t BAKED_COLOR_PACKAGE[] = {
+static constexpr uint8_t BAKED_COLOR[] = {
     #include "generated/material/bakedColor.inc"
 };
 
+static auto BAKED_COLOR_PACKAGE = Material::BufferDescriptor(BAKED_COLOR, sizeof(BAKED_COLOR));
```

Comment:
why this change? I don't think we want that. No reason to go through the BufferDescriptor -- which does heap allocation, etc...